.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: python
+
+python:
+  - 3.6
+
+dist: xenial
+sudo: false
+
+cache: pip
+
+install:
+  - pip install --upgrade pycodestyle
+
+script:
+  - pycodestyle nyaa/ --show-source --max-line-length=100
```

Comment:
Lint script runs `autopep8` 
I didn't like the report style provided by it.
Also it is based on pycodestyle with some minor additions AFAIK.